src/collectors/systemd-units.plugin/plugin_systemd_units.c

```diff
@@ -2177,4 +2187,71 @@ void function_systemd_units(
     systemd_units_free_all(base);
 }
 
-#endif // ENABLE_SYSTEMD_DBUS
+static bool plugin_should_exit = false;
+
+int main(int argc __maybe_unused, char **argv __maybe_unused)
```

Comment:
`__maybe_unused` doesn't make sense here, either because you're using it as an argument or because Netdata passes `update_every` as an argument to plugins, expecting it to be used by the plugin.